src/services/analyticsTracking.js

```diff
@@ -97,7 +97,7 @@ const setUserDimensions = (userId, role, groups) => {
   }
 
   if ((groups && groups.length) || (GAState.egoGroups && GAState.egoGroups.length)) {
-    let _groups = JSON.stringify({ egoGroups: groups[0] || GAState.egoGroups[0] });
+    let _groups = JSON.stringify({ egoGroups: groups ? groups[0] : GAState.egoGroups[0] });
```

Comment:
groups was undefined here. It's an OR condition so that can happen...

Reply:
What if just set the GAState.egoGroups default of `[" "]` that way it will pass through and we don't have to worry about doing so many conditional? 

I've been meaning to refactor this whole service with better patterns but haven't had time yet.

Reply:
It's actually `groups` that's undefined so `groups[0]` would throw an error. I'm not familiar with the rest of this so did this just to fix that one error without touching the rest and break things :P